### PR TITLE
Geojson serializer tests

### PR DESCRIPTION
**Pull request by leplatrem**
- [x] Fix with empty collections
- [x] Fix with JSONP
- [x] Add tests for GeoJSON field type (will fail with current code)

### CODE REVIEW

**Comment by leplatrem** [comment]
Tests fail mysteriously on python 2.6 and pypy :)

**Comment by leplatrem** [comment]
Ready! @ametaireau @Natim pls merge after quick overlook!

File: tox.ini [hunk 1]
```diff
@@ -13,6 +13,15 @@ deps =
...
 install_command = pip install --process-dependency-links --pre {opts} {packages}
 
+[testenv:py26]
+deps =
+    coverage
+    nose
+    webtest
+    unittest2
+    mock
+    ordereddict
```
Comment (almet) [inline tox.ini]: we probably want to put ordereddict in the dev-requirements as well

Comment (Natim) [inline tox.ini]: Or better add it in the REQUIREMENTS variable of setup.py only if python < 2.7
Actually using d2to1 it doesn't seams possible.

**Comment by leplatrem** [comment]
Should be better now :) Thanks for spotting this, it was shitty

**Comment by almet** [comment]
looks good to me :)

**Pull request merged**